introduction.md

```diff
@@ -0,0 +1 @@
+Hi, my name is Vitor and I'm doing the Mastering GitHub course from CodeSchool.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.